standup_notes/__main__.py

```diff
@@ -203,6 +208,34 @@ def delete_notes(date_to_delete):
         print("No files to be deleted")
 
 
+def post_note(day: date):
+    print("Posting your note")
+    date_of_note = "Date: " + day.strftime("%m/%d/%Y") + " \n"
+    link = input("Please put in the connector link: ")
```

Comment:
Could you add some instructions, maybe in the README, about what a connector link is?